diff --git a/GobiNet/Makefile b/GobiNet/Makefile
index cc55658..8cb8997 100644
--- a/GobiNet/Makefile
+++ b/GobiNet/Makefile
@@ -70,6 +70,6 @@ install: all
 	depmod
 
 clean:
-	rm -rf *.o *~ core .depend .*.cmd *.ko *.mod.c .tmp_versions Module.* modules.order .cache.mk *.o.ur-safe
+	rm -rf *.o *~ core .depend .*.cmd *.ko *.mod.c .tmp_versions Module.* modules.order .cache.mk *.o.ur-safe *.mod
 
 
diff --git a/GobiSerial/Makefile b/GobiSerial/Makefile
index 1349af8..2ddf72b 100644
--- a/GobiSerial/Makefile
+++ b/GobiSerial/Makefile
@@ -48,4 +48,4 @@ install: all
 	depmod
 
 clean:
-	rm -rf *.o *~ core .depend .*.cmd *.ko *.mod.c .tmp_versions Module.* modules.order .cache.mk *.o.ur-safe
+	rm -rf *.o *~ core .depend .*.cmd *.ko *.mod.c .tmp_versions Module.* modules.order .cache.mk *.o.ur-safe *.mod
